src/applications/vaos/api/mockApi.js

```diff
@@ -0,0 +1,11 @@
+export function getUserAppointmentsSummary() {
```

Comment:
Sorry, I think I confused things by adding this endpoint to the first pass at the api design. There's no equivalent to this in the current api, so we probably won't be able to create it. I think this page would need to fetch the whole request and appointments lists. I think it'd be better to mock those out than this summary endpoint. 

I think you could make those changes in this PR, or we could make a ticket to come back to it after my PR and the confirmed appointments list ticket (at which point, we'll have Redux boilerplate for both api requests).